interface.html

```diff
@@ -35,7 +35,9 @@
   </div>
 
   <div id="screenSection" class="section">
+    {{#unless options.hideAction}}
```

Comment:
To remove the big header, we don't need it when preselecting an action.
See requirements.